incubator/istio/templates/custom-resources.yaml

```diff
@@ -0,0 +1,383 @@
+{{- if and .Values.istio.install (not .Release.IsInstall) -}}
```

Comment:
Best practice is to have one file per resource. Maybe you could put them in a subfolder `custom-resources`. Would be much clearer.